styles/main/style.css

```diff
@@ -303,6 +303,60 @@ html {
   justify-content: space-between;
 }
 
+/* Testimonial section */
+.testimonial {
+  margin-top: 280px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.testimonial__title {
+  font-size: 54px;
+  font-weight: 500;
+}
+.testimonial__swiper {
+  margin-top: 100px;
+  width: 100%;
+}
+.testimonial__content {
+  display: flex;
+  gap: 100px;
+}
+.testimonial__comment {
+  font-size: 40px;
+}
+.testimonial__text {
+  display: flex;
+  flex-direction: column;
+  gap: 95px;
+}
+.testimonial__name {
+  font-size: 28px;
+  font-weight: 500;
+}
+.testimonial__desc {
+  font-size: 20px;
+  margin-top: 12px;
+}
+.testimonial__buttons {
+  display: flex;
+}
+.testimonial__boxes__pagination {
+  display: flex;
+  align-items: center;
+  gap: 24px;
+}
+.testimonial__quatation.right {
+  float: right;
+}
```

Comment:
`float: right` works, but I will give you a trick you might already know. You can say: `margin-left: auto`. In this case you also have to set `display` to `block`.

Don't have to change it, I'm just letting you know.

Reply:
You can read more here: https://css-tricks.com/is-css-float-deprecated/